src/main/java/racingcar/ui/Reader.java

```diff
@@ -0,0 +1,16 @@
+package racingcar.ui;
+
+public class Reader {
+
+  private static final java.util.Scanner scanner = new java.util.Scanner(System.in);
```

Comment:
이 부분도 잘 정의해주셨는데 convention 한번만 보고 가시면 더 좋을 것 같습니다 🙇 
```suggestion
  private static final java.util.Scanner SCANNER = new java.util.Scanner(System.in);
```